iam-login-service/src/main/java/it/infn/mw/iam/core/oauth/profile/keycloak/KeycloakProfileAccessTokenBuilder.java

```diff
@@ -67,6 +71,8 @@ public JWTClaimsSet buildAccessToken(OAuth2AccessTokenEntity token,
       }
     }
 
+    addAcrClaimIfNeeded(builder, authentication);
```

Comment:
if all the profiles are calling this logic, we could simply include it inside the parent constructor, or not?